src/microbe_stage/MicrobeAI.cs

```diff
@@ -67,21 +76,42 @@ public MicrobeAI(Microbe microbe)
         compoundsSearchWeights = new Dictionary<Compound, float>();
     }
 
-    private float SpeciesAggression => microbe.Species.Behaviour.Aggression;
-    private float SpeciesFear => microbe.Species.Behaviour.Fear;
-    private float SpeciesActivity => microbe.Species.Behaviour.Activity;
+    private float SpeciesAggression => microbe.Species.Behaviour.Aggression *
+        (receivedCommand == MicrobeSignalCommand.BecomeAggressive ? 1.5f : 1.0f);
+
+    private float SpeciesFear => microbe.Species.Behaviour.Fear *
+        (receivedCommand == MicrobeSignalCommand.BecomeAggressive ? 0.75f : 1.0f);
+
+    private float SpeciesActivity => microbe.Species.Behaviour.Activity *
+        (receivedCommand == MicrobeSignalCommand.BecomeAggressive ? 1.25f : 1.0f);
```

Comment:
While I'm still not a big fan of such strong signaling agents, if you want to go with the paradigm of "when under a command, the microbe ALWAYS follows it", my guess is you will need to go way bigger with your multipliers. If a player has a non-aggressive and fearful species, it's not going to attack something bigger than itself unless you multiply and divide by 10 or more. 

Even then, keep in mind that the minimum of each stat is 0, so if a player slides all the way to the left signaling agents won't matter. I don't know if that's intended functionality.